tests/sentry/relay/config/test_metric_extraction.py

```diff
@@ -122,14 +122,17 @@ def test_get_metric_extraction_config_single_alert(default_project: Project) ->
         config = get_metric_extraction_config(default_project)
 
         assert config
-        assert len(config["metrics"]) == 1
```

Comment:
In this PR, I'm removing all instances of checking for the length of the metrics if we check for the contents of it right after.